index.html

```diff
@@ -96,6 +96,37 @@ <h2 class="recent-work-container__header">My Recent Work</h2>
           </div>   
     </section>
 
+    <section class="testimonials">
+      <h2 class="testimonials__header">Testimonals</h2>
+
+      <p class="testimonials__subheader">People I've worked with have said some nice things...</p>
+      <div class="testimonials__carousel testimonials__carousel--selected">
+        <img src="./images/pascal.png" alt="picture of my employer" class="testimonials__carousel__picture">
+
+        <p class="testimonials__carousel__description">“Matt was a real pleasure to work with and we look forward to working with him again.
+          He's definitely the kind of designer you can trust with a project from start to finish.”</p>
+      
+        <div class="testimonials__carousel__name">Pascal Trembly</div>
+
+        <div class="testimonials__carousel__signature">Creative lead, Good Kind</div>
+      </div>
+      <div class="testimonials__carousel">
+        <img src="./images/alvin.png" alt="picture of my employer" class="testimonials__carousel__picture">
```

Comment:
this is an ok solution but if you want to go the extra step why not iterate over and array of objects containing the config for the carousel? so its more easily upgradable for the future